### ISSUE TITLE
[BUG] Theme Icon Is Not Changing When In Dark Mode

### ISSUE THREAD

**Issue by Karthik-315** [subhoghoshX/laureate#57]
The theme icon is the same for dark and light modes. I think it would be better to add different icons for light and dark.

Also, as an added feature we can introduce persistence by adding the theme preference to `localStorage`, thereby loading the user-preferred theme when the page loads.

I can work on the above two features. Let me know your thoughts @subhoghoshX .

**Comment by subhoghoshX** [comment]
Yeah, I just thought a glowing moon icon might be more than enough, similar to https://play.tailwindcss.com/. About the localStorage thing, feel free to do it. I read zustand has a persistent function that can save states in local storage but didn't have time to look into it. You can look into that too.

**Comment by Karthik-315** [comment]
Awesome,  I'll work on implementing persistence

**Comment by subhoghoshX** [comment]
Cool, don't change the icon now. I've installed heroicons, when I push the commit, you'll be just able to import the light and dark icons.

Also, check out `immer` in zustand. Andrej said that might be what I'm looking for. I don't know. I didn't have time to look into that either.

**Comment by Karthik-315** [comment]
Cool. I will work only on the persistence for now, for the icons, let me know. I've looked into the `persist` function, I'll also take a look at `immer` too.

**Comment by Karthik-315** [comment]
Closing the issue as the relevant PR #58  is approved & merged.

**Comment by subhoghoshX** [comment]
> Closing the issue as the relevant PR #58 is approved & merged.

What about the icon change?

**Comment by Karthik-315** [comment]
> > Closing the issue as the relevant PR #58 is approved & merged.
> 
> What about the icon change?

Oh yeah, totally forgot about it. I'll add the **SunIcon** from the heroicons pack you added, and send add a PR. 

Reopening the issue.
